### PR TITLE
Support for eZ Platform 2.5.0

### PR DESCRIPTION
**Pull request by pspanja**

### CODE REVIEW

File: lib/Core/Repository/Aggregate/Repository.php [hunk 1]
```diff
@@ -64,9 +64,12 @@ public function setCurrentUser(UserReference $user)
         return $this->ezRepository->setCurrentUser($user);
     }
 
-    public function sudo(Closure $callback)
+    public function sudo(callable $callback, RepositoryInterface $outerRepository = null)
```
Comment (emodric) [inline lib/Core/Repository/Aggregate/Repository.php:67]: Does changing `Closure` to `callable` make this incompatible with previous versions of eZ kernel?

Comment (pspanja) [reply lib/Core/Repository/Aggregate/Repository.php:67]: No; previously the method was not in the interface and `callable` type hint will allow `Closure` instance. I updated the test setup to allow testing integration with different versions of `ezpublish-kernel`, everything works OK.

Comment (emodric) [reply lib/Core/Repository/Aggregate/Repository.php:67]: 👍

**Review by emodric — APPROVED** [review]

**Comment by pspanja** [comment]
Thanks for the review!

**Pull request merged**